<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qt_c02</class>
 <widget class="QDialog" name="qt_c02">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>726</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>录屏软件</string>
  </property>
  <widget class="QPushButton" name="btn_start">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>70</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>开始录屏</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭程序</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="t_directory">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文件夹路径</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_select">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_open_path">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>110</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>打开下载目录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_countDown">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>190</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>倒计时</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_digit">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>130</y>
     <width>191</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_exit</sender>
   <signal>clicked()</signal>
   <receiver>qt_c02</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
